prql-compiler/src/lib.rs

```diff
@@ -54,7 +54,6 @@ mod test {
     use insta::{assert_display_snapshot, assert_snapshot};
 
     #[test]
-    #[ignore]
```

Comment:
Nice! We can run with `--ignored` to get all the tests (though I'm not sure it's like python which will flag `xfail` tests that pass)